tests/testsuite/env.rs

```diff
@@ -721,3 +721,62 @@ fn test_parse_uint_default() {
     let config: TestUint = config.try_deserialize().unwrap();
     assert_eq!(config.int_val, 42);
 }
+
+#[cfg(test)]
```

Comment:
all 3 tests are failing before the fixing commit :+1:

Reply:
Every commit should be atomic which includes passing tests.

In this case, the change is simple, just adding a `#[should_panic]` to each test (preferably with the message set so we know it isn't an assert).  We specifically encourage this splitting of commits in our [contributor guide](https://github.com/rust-cli/config-rs/blob/main/CONTRIBUTING.md#process).

Reply:
Cool - I read that part in the contributor guide but didn't think of `should_panic`. Practically like xfailing the test. Makes sense, added it.